Update netty-tcnative-boringssl to 2.0.8

**Problem description**
The `netty-tcnative-boringssl-static` dependency (`2.0.6`) has a bug on Windows that is fixed by the latest version (`2.0.8`).   The bugfix commit is [here](https://github.com/netty/netty-tcnative/commit/d7fd591b077e3ce1e906c81d48a69d713598aa39).

It manifests only when the client dependencies are shaded, as an exception when setting up the GRPC SSL context:
```
Exception in thread "main" java.lang.IllegalArgumentException: ALPN is not configured properly. See https://github.com/grpc/grpc-java/blob/master/SECURITY.md#troubleshooting for more information.
	at io.grpc.netty.shaded.io.grpc.netty.GrpcSslContexts.selectApplicationProtocolConfig(GrpcSslContexts.java:163)
	at io.pravega.shaded.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:136)
	at io.pravega.shaded.io.grpc.netty.GrpcSslContexts.configure(GrpcSslContexts.java:124)
	at io.pravega.shaded.io.grpc.netty.GrpcSslContexts.forClient(GrpcSslContexts.java:94)
	at io.pravega.client.stream.impl.ControllerImpl.<init>(ControllerImpl.java:150)
	at io.pravega.client.stream.impl.ControllerImpl.<init>(ControllerImpl.java:123)
	at io.pravega.client.admin.impl.StreamManagerImpl.<init>(StreamManagerImpl.java:39)
	at io.pravega.client.admin.StreamManager.create(StreamManager.java:39)
	at io.pravega.client.admin.StreamManager.create(StreamManager.java:29)
	at io.pravega.connectors.flink.util.FlinkPravegaParams.createStream(FlinkPravegaParams.java:144)
	at io.pravega.connectors.flink.util.FlinkPravegaParams.createStream(FlinkPravegaParams.java:140)
	at io.pravega.examples.flink.wordcount.WordCountWriter.main(WordCountWriter.java:66)
Caused by: java.lang.ClassNotFoundException: org/eclipse/jetty/alpn/ALPN
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at io.pravega.shaded.io.grpc.netty.JettyTlsUtil.isJettyAlpnConfigured(JettyTlsUtil.java:34)
	at io.pravega.shaded.io.grpc.netty.GrpcSslContexts.selectApplicationProtocolConfig(GrpcSslContexts.java:153)
	... 11 more
```

The above exception is misleading as it relates to some fallback logic.  The root cause is that boringssl couldn't be loaded.

**Suggestions for an improvement**
Please update to `io.netty:netty-tcnative-boringssl-static:2.0.8.Final`.    Seems to work.

--- a/build.gradle
+++ b/build.gradle
@@ -422,8 +422,7 @@ project('shared:controller-api') {
         compile "io.grpc:grpc-all:" + grpcVersion
         compile "io.grpc:grpc-protobuf:" + grpcVersion
         compile "io.grpc:grpc-stub:" + grpcVersion
-        // https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static
-        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.6.Final'
+        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: nettyBoringSSLVersion
     }
 
     javadoc {
@@ -488,8 +487,7 @@ project('controller') {
         testCompile project(':test:testcommon')
         testCompile group: 'org.apache.curator', name: 'curator-test', version: apacheCuratorVersion
         compile 'io.jsonwebtoken:jjwt:0.9.0'
-        // https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static
-        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.6.Final'
+        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: nettyBoringSSLVersion
 
     }
 
@@ -549,8 +547,7 @@ project('standalone') {
         compile group: 'org.apache.hadoop', name: 'hadoop-common', version: hadoopVersion, withoutLogger
         compile group: 'org.apache.hadoop', name: 'hadoop-hdfs', version: hadoopVersion, withoutLogger
         compile group: 'org.apache.hadoop', name: 'hadoop-minicluster', version: hadoopVersion, withoutLogger
-        // https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static
-        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.6.Final'
+        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: nettyBoringSSLVersion
 
         testCompile project(':test:testcommon')
         testCompile project(path:':common', configuration:'testRuntime')

--- a/gradle.properties
+++ b/gradle.properties
@@ -38,6 +38,7 @@ metricsGangliaVersion=1.0.10
 mockitoVersion=2.10.0
 #Check issue in https://github.com/pravega/pravega/pull/2146 before when upgrading netty
 nettyVersion=4.1.16.Final
+nettyBoringSSLVersion=2.0.8.Final
 protobufGradlePlugin=0.8.3
 protobufProtocVersion=3.3.0
 qosLogbackVersion=1.2.3

